src/routes/sessions.ts

```diff
@@ -0,0 +1,56 @@
+import { CodeBox } from "@ory/elements-markup"
+import {
+  defaultConfig,
+  requireAuth,
+  RouteCreator,
+  RouteRegistrator,
+} from "../pkg"
+import { navigationMenu } from "../pkg/ui"
+
+export const createSessionsRoute: RouteCreator =
+  (createHelpers) => async (req, res) => {
+    const { sdk } = createHelpers(req)
+    const session = req.session
+
+    // Create a logout URL
+    const logoutUrl =
+      (
+        await sdk
+          .createSelfServiceLogoutFlowUrlForBrowsers(req.header("cookie"))
+          .catch(() => ({ data: { logout_url: "" } }))
+      ).data.logout_url || ""
+
+    res.render("session", {
+      layout: "welcome",
+      traits: {
+        ...session?.identity.traits,
+        ...{ created_at: session?.identity.created_at || "" },
```

Comment:
This doesn't need to be spread, right?

Reply:
Ah yes, you're right. it was just remnants of refactoring :)